lib/git.js

```diff
@@ -4,7 +4,7 @@ var Bluebird = require('bluebird');
 var CP       = Bluebird.promisifyAll(require('child_process'));
 
 var SEPARATOR      = '===END===';
-var COMMIT_PATTERN = /^(\w*)(?:\(([^)]*?)\)|):(.*?(?:\[([^\]]+?)\]|))\s*$/;
+var COMMIT_PATTERN = /^([^)]*)(?:\(([^)]*?)\)|):(.*?(?:\[([^\]]+?)\]|))\s*$/;
```

Comment:
This change is needed to make sure unknown commit types like `run-fix(core): Implement something` works.